### PR TITLE
hw-mgmt: global wp: Extend global write protection script

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
Add more logic for global write protection handling.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>

### CODE REVIEW

**Comment by vadimp-nvidia** [comment]
RERUN

File: usr/usr/bin/hw-management-global-wp.sh [hunk 1]
```diff
@@ -86,15 +109,38 @@ Usage: $(basename "$0") [Options]
...
+		log_info "Wrong command format."
+		echo "$__usage"
+		exit 1
+	fi
+
 	lock_service_state_change
-	do_global_wp_release_restore "$ACTION"
+	# Only one Global WP process can be activated.
+	if [ -f /var/run/hw-management-global-wp.pid ]; then
+		log_info "Global WP process is already rinning."
```
Comment (felixradensky) [inline usr/usr/bin/hw-management-global-wp.sh:137]: Typo in this line

**Comment by vadimp-nvidia** [comment]
RERUN

File: usr/usr/bin/hw-management-global-wp.sh (hunk 1 above)
Comment (vadimp-nvidia) [reply usr/usr/bin/hw-management-global-wp.sh]: Updated, thanks.

**Review by felixradensky — APPROVED** [review]

**Pull request merged**